src/02-data-types.js

```diff
@@ -79,7 +93,13 @@ function convertToWhisperText(text) {}
  *  checkIfCharacterIsInString("hello there", "a");
  *  //> false
  */
-function checkIfCharacterIsInString(text, character) {}
+function checkIfCharacterIsInString(text, character) {
+  
+  if(text.indexOf(character) == -1){
```

Comment:
Remember comparisons evaluate to booleans so we can just `return text.indexOf(character) !== -1`